tests/test_bugzilla.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import bugzilla
+
+
+class BugzillaTest(unittest.TestCase):
+
+    def test_get_signatures_from_bug(self):
+        crash_signatures = [u'0x0 | idmcchandler7_64.dll@0x238bf',
```

Comment:
We need to fix a bug in bugzilla.py, as this should return `@0x0 | idmcchandler7_64.dll@0x238bf`, etc..
I'll merge this for now, and file a new issue for the bug.